### ISSUE TITLE
Pressure control, profiling and preinfusion

### ISSUE THREAD

**Issue by greymda** [brycesub/silvia-pi#11]
any chance you are interested in adding these features?
now that the Raspberry Pi Zero W is released, i really am looking into fitting a smarter PID into my Silvia.

**Comment by brycesub** [comment]
Yes, I'm interested in adding this, or would be happy to receive a code contribution from you or someone else.  It'll require another relay, but all other hardware is the same.  The RPiZW looks awesome, hope this drives more adoption of projects like this.

**Comment by obiben** [comment]
Pre-infusion can be easily done by wiring the 3-way valve to a separate switch than the pump.
